forc-plugins/forc-client/tests/deploy.rs

```diff
@@ -1074,32 +1076,32 @@ async fn deployed_predicate_call() {
     wallet_unlocked
         .transfer(
             predicate.address(),
-            500,
-            base_asset_id,
+            2000,
```

Comment:
nit: It would be great to add a comment here explaining why the balance is what it is.